src/util.js

```diff
@@ -0,0 +1,10 @@
+export function dequote (input) {
+  input = input.trim()
+
+  const singleQuote = input.startsWith('\'') && input.endsWith('\'')
```

Comment:
Strings are immutable... Since this is for input in both cases ticks won't be used -- i'm going to avoid the regex for the sake of being explicit 😜